scalafmt-core/shared/src/main/scala/org/scalafmt/config/Spaces.scala

```diff
@@ -1,14 +1,18 @@
 package org.scalafmt.config
 
 import metaconfig._
-import org.scalafmt.config.SpaceBeforeContextBound.Never
```

Comment:
good idea removing the nested import but please don't rename below, just import as-is.